### PR TITLE
[Feature]: realign the count of cards in the deck

### LINKED ISSUE #231: [Feature]: realign the count of cards in the deck so it doesn’t touch the edge of the deck
### Feature Summary

The deck has a text blurb with the count of how many cards are in it, but it is awkwardly placed all the way at the bottom so it’s flush with the bottom of the deck. We should move it slightly upwards so there is a margin between the text and the bottom edge of the dexk

### Detailed Description

The (1) in the picture should be moved slightly upwards.

### PR DESCRIPTION
**Pull request by gulyapulya**
## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change

Fixes #231 

Before the deck's count was positioned at the very bottom of the deck and now it's been moved slightly up. 
 
Before:
image

After:
image

### CODE REVIEW

File: client/js/views/GameView.vue [hunk 1]
```diff
@@ -1217,6 +1217,7 @@ export default {
       position: absolute;
     }
     .c-deck-count {
+      margin: 6px;
```
Comment (itsalaidbacklife) [inline client/js/views/GameView.vue:1220]: We generally use multiples of 8px so let's use 8 instead of 6
```suggestion
      margin: 8px;
```

**Review by itsalaidbacklife — APPROVED** [review]
Looks good! I'm requesting that you increase the margin to 8px from 6, and could you also please increment the `version` field in package.json to 3.3.2? We try to increase the version every time we merge/deploy

**Comment by gulyapulya** [comment]
> Looks good! I'm requesting that you increase the margin to 8px from 6, and could you also please pull from main and then increment the `version` field in package.json to 3.3.3? We try to increase the version every time we merge/deploy

For sure, thanks for the review and suggestions! Will do now.

**Comment by gulyapulya** [comment]
@itsalaidbacklife, should be ready now! 😊
Including the latest commits in the main branch and patching the version to 3.3.3, margin size updated.

**Review by itsalaidbacklife — APPROVED** [review]
Great work; thanks for your contribution! Please feel free to pick up any other issue that interests you 😃

**Pull request merged**